src/main/java/sk/durixx/library/open/LibraryApi.java

```diff
@@ -0,0 +1,44 @@
+package sk.durixx.library.open;
+
+import sk.durixx.library.model.dto.LibraryDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import sk.durixx.library.domain.LibraryService;
+import sk.durixx.library.model.entity.Library;
+
+import java.util.List;
+
+
+@Component
+public class LibraryApi {
+
+    @Autowired
+    LibraryService libraryService;
+
+    public void createNewLibrary(LibraryDto library) {
+        libraryService.createNewLibrary(library);
+    }
+
+    public LibraryDto readLibrary(Long id) {
+        return LibraryDto.fromLibrary(libraryService.readLibrary(id));
+    }
+
+    public void updateLibrary(LibraryDto library) {
+        libraryService.updateLibrary(library);
```

Comment:
dont pass dto object to service instead use entity

Reply:
fixed in Patch for library #6